doc/get-set-friendlyname-mac.md

```diff
@@ -5,24 +5,26 @@ Friendly name and MAC address can be set and retrieved for each camera added to
 
 ## Preset FriendlyName
 `FriendlyName` is an element in the `Onvif ProtocolProperties` device field. It is initialized to be empty or `<Manufacturer+Model>`
-if credentials are provided on discovery. The user can also pre-define this field in a camera.toml file.
+if credentials are provided on discovery. The user can also pre-define this field in a camera.yaml file.
 
 If you add pre-defined devices, set up the `FriendlyName` field as shown in the
-[camera.toml.example file](../cmd/res/devices/camera.toml.example).
+[camera.yaml.example file](../cmd/res/devices/camera.yaml.example).
 
-```toml
+```yaml
 # Pre-defined Devices
-[[DeviceList]]
-Name = "Camera001"
-ProfileName = "onvif-camera"
-Description = "onvif conformant camera"
-  [DeviceList.Protocols]
-    [DeviceList.Protocols.Onvif]
-    Address = "192.168.12.123"
-    Port = "80"
-    FriendlyName = "Home camera"
-    [DeviceList.Protocols.CustomMetadata]
-    Location = "Front door"
+DeviceList:
+  - Name: Camera001
+    ProfileName: onvif-camera
+    Description: onvif conformant camera
+    Protocols:
+      Onvif:
+        Address: 192.168.12.123
+        Port: '80'
+        FriendlyName: Home camera
+        MACAddress: 'aa:bb:cc:dd:ee:ff'
+      CustomMetadata:
+        Location: Front door
+        Color: Black and white
```

Comment:
Make YAML example match TOML example exactly. don't add/remove things. These are not meant to match the file data exactly.